CharacterCreation/03_Feats.txt

```diff
@@ -872,7 +872,32 @@ you are wielding an appropriate weapon.
 	unarmed targets).
 60:	+10% Base damage. Gain +0.5 Critical Hit range whenever you hit an enemy but
 	do not score a Critical Hit. This bonus resets if you score a Critical Hit, 
-	are stunned, or spend 1 round out of Melee Combat.
+	are stunned or knocked prone, or spend 1 round out of Melee Combat.
+
+== Thrown Expertise ==
+
+	Applies to throwing weapons and large-projectile weapons like bows, javelins, 
+and harpoon launchers that the user has proficiency with. Does not apply to Improvised 
+throwing weapons.
+
+0:	+5m to ranges R1 and R2.
+12:	May use Weapons - Melee in place of Perform (Dex) when making displays of 

```

Comment:
*Performance
I'm a little confused by having "Weapons - Melee" here instead of "Weapons - Thrown." Is that intentional?

Reply:
Yes, it was, but at the time I mistakenly thought we didn't have a Weapons - Thrown skill. I should have checked. Will fix.

Related: What do we use Weapons - Thrown for (besides this feat)? If not enough things that's a new issue.